lms/djangoapps/teams/static/teams/js/spec/views/teams_tab_spec.js

```diff
@@ -39,7 +57,8 @@ define([
 
         describe('Navigation', function () {
             it('does not render breadcrumbs for the top level tabs', function() {
-                var teamsTabView = createTeamsTabView();
+                var requests = AjaxHelpers.requests(this),
```

Comment:
I would prefer you not define a variable and include a comment. Otherwise someone else will just try to delete it when they are in the code sometime and notice an unused variable.

// Rendering the view triggers events, so Sinon needs to be in place.
AjaxHelpers.requests(this);